src/Components/EventDetails/EventDetails.js

```diff
@@ -12,13 +12,13 @@ import { getEvent } from '../../queries/index';
 export const EventDetails = ({loggedInUser, logoutUser}) => {
   const {id} = useParams();
 
+
   const { loading, error, data } = useQuery(getEvent, { variables: {
     id
   }, skip: !id });
-
   if (loading) return <p>Loading...</p>; 
   if (error) return <p>Error :</p>; 
-
+  console.log('data',data)
```

Comment:
I'm not too concerned cause I know we will get all the logs out when we are all done, but just marking this one!